docs/reference/commandline/plugin_enable.md

```diff
@@ -21,7 +21,8 @@ Usage:  docker plugin enable PLUGIN
 Enable a plugin
 
 Options:
-      --help   Print usage
+      --help          Print usage
+      --timeout int   HTTP client timeout (in seconds)
```

Comment:
Just discussed with @vdemeester that this should probably be a `duration`, to be consistent with other options.